src/Schema/Builder.php

```diff
@@ -188,34 +196,49 @@ public function getTables($schema = null)
         return $collections;
     }
 
-    /** @param  string|null $schema Database name */
-    public function getViews($schema = null)
+    private function collationToString(array $collation): string
     {
-        $db = $this->connection->getDatabase($schema);
-        $collections = [];
-
-        foreach ($db->listCollections() as $collectionInfo) {
-            $collectionName = $collectionInfo->getName();
-
-            // Skip normal type collection
-            if ($collectionInfo->getType() !== 'view') {
-                continue;
+        $map = [
+            'locale' => 'l',
+            'strength' => 's',
+            'caseLevel' => 'cl',
+            'caseFirst' => 'cf',
+            'numericOrdering' => 'no',
+            'alternate' => 'a',
+            'maxVariable' => 'mv',
+            'normalization' => 'n',
+            'backwards' => 'b',
+        ];
+
+        $parts = [];
+        foreach ($collation as $key => $value) {
+            if (array_key_exists($key, $map)) {
+                $shortKey = $map[$key];
+                $shortValue = is_bool($value) ? ($value ? '1' : '0') : $value;
+                $parts[] = $shortKey . '=' . $shortValue;
             }
-
-            $collections[] = [
-                'name' => $collectionName,
-                'schema' => $db->getDatabaseName(),
-                'schema_qualified_name' => $db->getDatabaseName() . '.' . $collectionName,
-                'size' => null,
-                'comment' => null,
-                'collation' => null,
-                'engine' => null,
-            ];
         }
 
-        usort($collections, fn ($a, $b) => $a['name'] <=> $b['name']);
+        return implode(';', $parts);
+    }
 
-        return $collections;
+    /** @param string|null $schema Database name */
+    public function getTables($schema = null)
+    {
+        $aggregateStats = function (Database $db, string $collectionName): array {
+            return $db->selectCollection($collectionName)->aggregate([
```

Comment:
We renamed this method to `getCollection`: https://github.com/mongodb/mongo-php-library/pull/1494
```suggestion
            return $db->getCollection($collectionName)->aggregate([
```